docs/src/main/sphinx/connector/base-jdbc.rst

```diff
@@ -0,0 +1,194 @@
+===================
+JDBC connector base
+===================
+
+A number of connectors use JDBC connections to the data source. Typically
```

Comment:
- use JDBC to connect to data sources
- - Typically,

Reply:
Done